index.html

```diff
@@ -204,48 +252,46 @@ <h2 class="about-text">
           like what you see and have a project you need to be coded, don’t
           hestiate to contact me.
         </p>
-        <a href="#" class="connect-bottom">LET'S CONNECT</a>
-        <ul class="social-media-button">
+        <a href="#" class="connect connect-bottom">LET'S CONNECT</a>
+        <ul class="social-media-icons social-media-button">
           <li>
             <a href="#" class="icons">
-              <img src="Socials/GitHub.png" alt="Github" />
+              <img src="Socials/Twitter.png" alt="Twitter logo" />
             </a>
           </li>
           <li>
             <a href="#" class="icons">
-              <img src="Socials/Linkedin.png" alt="Linkedin" />
+              <img src="Socials/Linkedin.png" alt="Linkedin logo" />
             </a>
           </li>
           <li>
             <a href="#" class="icons">
-              <img src="Socials/M.png" alt="Medium" />
+              <img src="Socials/M.png" alt="Medium logo" />
             </a>
           </li>
           <li>
             <a href="#" class="icons">
-              <img src="Socials/peacesign.png" alt="Angelist" />
+              <img src="Socials/GitHub.png" alt="Github logo" />
             </a>
           </li>
           <li>
             <a href="#" class="icons">
-              <img src="Socials/Twitter.png" alt="Twitter" />
+              <img src="Socials/peacesign.png" alt="Angelist logo" />
             </a>
           </li>
         </ul>
-        <button class="btn btn--resume" type="button">Get my Resume</button>
+        <button type="submit" class="btn btn--resume">Get my Resume</button>
       </div>
 
       <div class="skills-section">
         <ul class="tools">
           <li class="tabs">
             <h4 class="caption">Languages</h4>
-              <a href="#">
-                <img
-                  class="left-sidebar"
-                  alt="open-tab"
-                  src="arrow1.png"
-                />
-            </a>  
+            <img
+              class="left-sidebar"
+              alt="open-tab"
+              src="arrow1.png"
+            />
           </li>
         </ul>
         <ul class="side-language">
```

Comment:
Kindly note that the text of languages should be left-aligned as shown below 😃

Your design:
![image](https://user-images.githubusercontent.com/73714615/186906732-a36046a7-c5dd-486e-bec9-8491b18b306c.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186906765-20b42bd0-3172-4722-86d1-441029cf585e.png)